### PR TITLE
[satel] INT-RS support added.

### PR DESCRIPTION
**Pull request by druciak**
This PR allows to connect to Satel INT-RS via RS-232 interface. Besides that it introduces some other communication improvements and code refactoring around binding configuration.

### CODE REVIEW

File: bundles/binding/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/config/ConfigIterator.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	@Override
+	public String next() {
+		return this.configElements[idx++];
+	}
+
+	@Override
+	public void remove() {
+		// ignore
+	}
+}
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/config/ConfigIterator.java]: add newline

File: bundles/binding/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/protocol/IntRSModule.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2010-2015, openHAB.org and others.
+ * Copyright (c) 2010-2014, openHAB.org and others.
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/protocol/IntRSModule.java]: revert this change

**Comment by teichsta** [comment]
Hi @druciak thanks for this contribution. Please find some minor comments inline. Please squash your commits into again once you've incorporated my feedback. Thanks, Thomas E.-E.

**Comment by druciak** [comment]
Done. 
One question though. What does it change when you assign me to a PR? Are there any additional operations I can make or this is just a marker?

**Comment by teichsta** [comment]
> What does it change when you assign me to a PR? Are there any additional operations I can make or this is just a marker?

it's just a marker to say who's next in the workflow. Since i was done with the review i expected to incorporate my feedback :-)

Thanks btw!
